### ISSUE TITLE
color contrast issue in rule pages

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules-web#212]
The links to input aspects have insufficient contrast when hovered. Learn more about techniques under Accessibility mapping is also failing minimum contrast. It turns light blue on hover.
